### DISCUSSION TITLE
태그를 필수로 할것인가?

### DISCUSSION THREAD

**Discussion by airman5573** [woowacourse-teams/2022-moamoa#158] [질문] [answered]
현재 카테고리 리스트

- FE/BE (area)
- 기수 (generation)
- 태그 (js, ts, java, spring, react ...)

어떤걸 필수 태그로 할것인가?

현재 의견 : generation / tag 만 필수로  &  area는 optional

**Comment by nan-noo** [comment, marked as answer]
subject를 필수로 하는 게 좋을 것 같습니다 :D 
etc 필드를 추가해서..!
